test/cppsuite/test_harness/timestamp_manager.cpp

```diff
@@ -126,6 +126,16 @@ timestamp_manager::get_next_ts()
     return (current_time);
 }
 
+wt_timestamp_t
+timestamp_manager::get_random_ts()
+{
+    wt_timestamp_t ts = random_generator::instance().generate_integer(
+      (_oldest_ts >> 32), (this->get_next_ts() >> 32));
```

Comment:
I think here you can just use get_time_now_s instead of get_next_ts? then we don't need to fuss around with left and right shift.